webapp/src/main/webapp/src/app/sender/archive/search-event.ts

```diff
@@ -0,0 +1,9 @@
+export class SearchEvent {
+
+	startDate?: string = '';
+
+	endDate?: string = '';
+
+	tableName?: string = '';
```

Comment:
I thought you don't support search by tableName, in fact do you need this `SearchEvent` class, what you need is a `SearchAction`